### DISCUSSION TITLE
SEB double authenthication

### DISCUSSION THREAD

**Discussion by sgrandh3** [SafeExamBrowser/seb-win-refactoring#590] [Q&A]
For testing, students need to sign in twice, meaning 2 duo codes Is there a way to not have that? They already sign in to moodle and launch the exam from moodle

**Comment by dbuechel** [comment]
I am not entirely sure, but the Moodle Deeper Integration with SEB may help with this issue. I do however know that other LMS append a sign-in token to the end of the start URL in order to prevent users from having to sign-in twice.

**Reply by sgrandh3** [reply]
Thank you @dbuechel for your insights

**Comment by danschlet** [comment]
The Moodle SEB integration currently doesn't support such a sign-in token for automatic sign-in in SEB after starting the Moodle quiz in another browser. I would try to place a feature request at moodle.org, we would definitely support that, but we can't do it on our own.

**Reply by sgrandh3** [reply]
Thank you @danschlet for your quick reply and guidance.
